build/buildpipeline/pipeline.groovy

```diff
@@ -0,0 +1,11 @@
+import org.dotnet.ci.pipelines.Pipeline
+
+def windowsPipeline = Pipeline.createPipeline(this, 'windows.groovy')
+def linuxPipeline = Pipeline.createPipeline(this, 'linux.groovy')
+def osxPipeline = Pipeline.createPipeline(this, 'osx.groovy')
+String configuration = 'Release'
+
+[windowsPipeline, linuxPipeline, osxPipeline].each { pipeline ->
+    pipeline.triggerPipelineOnEveryGithubPR(['Configuration':configuration])
```

Comment:
Missing an argument. This method requires you specify a trigger name. Example:
```groovy
pipeline.triggerPipelineOnEveryGithubPR("Windows ${configuration} x64 Build", ['Configuration':configuration])
```

Reply:
Ok. Do you need the empty groovy files to be present before it can build?

Reply:
I think so. I added sub files to the dev branch of aspnet-docker first to get this rolling. That triggers the "generator" build. Example https://ci3.dot.net/job/aspnet_aspnet-docker/job/dev/job/generator/

Reply:
The generator reads the job declarations upon each commit.  The job definition (code that actually runs) is executed based on the code in the PR/branch (for commit jobs)

Reply:
Ok. Pushed an update